bitnami/mongodb/values-production.yaml

```diff
@@ -354,8 +428,27 @@ sidecars: {}
 ## extraVolumes:
 ##   - name: extras
 ##     emptyDir: {}
-extraVolumeMounts: []
-extraVolumes: []
+extraVolumeMounts:
+  - name: certs
+    mountPath: /certs
+
+extraVolumes:
+  - name: certs
+    emptyDir: {}
+  - name: certs-volume
+    secret:
+      secretName: ca-clients
+      items:
+      - key: mongodb-ca-cert
+        path: mongodb-ca-cert
+        mode: 511
+      - key: mongodb-ca-key
+        path: mongodb-ca-key
+        mode: 511
+
+
+#extraVolumeMounts: []
+#extraVolumes: []
```

Comment:
We should not be setting this

Reply:
What will be the workaround for this then? We have to set this in order for the TLS volumes and volume mounts to be created which will contain the certs for accessing the cluster.

Reply:
We can also try setting this in the statefulset.yaml file so its not necessarily needed to be set int he values-production yaml file